src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -5973,12 +5984,16 @@ def create_vpn_connection(cmd, resource_group_name, connection_name, vnet_gatewa
     deployment_name = 'vpn_connection_deploy_' + random_string(32)
     client = get_mgmt_service_client(cmd.cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES).deployments
     properties = DeploymentProperties(template=template, parameters=parameters, mode='incremental')
+    deployment = Deployment(properties=properties)
```

Comment:
Is it a breaking change?

Reply:
Yes, it is a breaking change of SDK, and we did made some compatibility logic to ensure that the CLI will not be broken